src/platform/qt/Window.cpp

```diff
@@ -1813,6 +1813,11 @@ void Window::setupOptions() {
 		reloadConfig();
 	}, this);
 
+	ConfigOption* rewindRatio = m_config->addOption("rewindRatio");
```

Comment:
The historical name for this setting has been `rewindBufferInterval`, so I'd recommend using that just in case someone has a config file that old.

```suggestion
	ConfigOption* rewindRatio = m_config->addOption("rewindBufferInterval");
```